cp3pt0-deployment/setup_singleton.sh

```diff
@@ -227,4 +235,16 @@ function get_and_validate_arguments() {
     get_control_namespace
 }
 
+function accept_license() {
+    if [[ $LICENSE_ACCEPT == "true" ]]; then
```

Comment:
```suggestion
    if [ $LICENSE_ACCEPT -ne 1 ]; then
        return
    fi
```

invert it so that function returns early if license was not accepted

Reply:
move this check to prereq and error out of script